src/cmake/common-settings.cmake

```diff
@@ -233,8 +233,7 @@ if("${CMAKE_BUILD_TYPE}" STREQUAL "release" OR "${CMAKE_BUILD_TYPE}" STREQUAL "t
 	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS_RELEASE}")
 	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS_RELEASE}")
 else()
-	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS_DEBUG}")
-	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS_DEBUG}")
-	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS_DEBUG}")
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS_DEBUG} ${COVERAGE_CXX_FLAGS}")
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS_DEBUG} ${COVERAGE_CXX_FLAGS}")
+	set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS_DEBUG} ${COVERAGE_LINKER_FLAGS}")
```

Comment:
Remove `${CMAKE_EXE_LINKER_FLAGS_DEBUG}` and `${CMAKE_EXE_LINKER_FLAGS_RELEASE}` if they're unused